src/index.js

```diff
@@ -73,6 +73,14 @@ function initPolyfill() {
       "Error installing ScrollTimeline polyfill: could not attach WAAPI's getAnimations to document"
     );
   }
+
+  const origSupports = CSS.supports;
+  CSS.supports = (ident) => {
+    if (ident.includes('animation-timeline:')) {
```

Comment:
There's probably no reason to check if the string includes animation-timeline, if we're only replacing animation-timeline. Also the includes check includes the trailing colon but the replace doesn't, is this a typo?

Reply:
Removing check …